### ISSUE TITLE
manpage -- args are shown as emdashes

### ISSUE THREAD

**Issue by alxndrsn** [koalaman/shellcheck#2699]
```
$ shellcheck --version
ShellCheck - shell script analysis tool
version: 0.7.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

On Ubuntu, `man shellcheck` seems to have converted double-dash (`--`) args into em-dashes (or similar):

```
       –list-optional
              Output a list of known optional checks.  These can be enabled with -o flags or enable directives.
```

This seems incorrect:

```
$ shellcheck –list-optional
–list-optional: –list-optional: openBinaryFile: does not exist (No such file or directory)
```

vs double-dash:

```
$ shellcheck --list-optional
name:    add-default-case
desc:    Suggest adding a default case in `case` statements
example: case $? in 0) echo 'Success';; esac
fix:     case $? in 0) echo 'Success';; *) echo 'Fail' ;; esac
...
```

**Comment by alxndrsn** [comment]
Maybe a duplicate of https://github.com/koalaman/shellcheck/issues/1392

**Comment by brother** [comment]
The commit for #1392 ([1397](https://github.com/koalaman/shellcheck/pull/1397)) was released in 0.6.0 and as long as the ubuntu (actually debian, no special handling in ubuntu afaict) package is not doing anything strange should already be included in your 0.7.0. 

In debian with 0.9.0 the man page does not have this problem. Could it be some "smart" terminal emulator involverd? The code and packages seems to not be broken at least.

**Comment by alxndrsn** [comment]
>  Could it be some "smart" terminal emulator involverd?

I don't think so... I've just tested as well with `xterm` and `st`, and they have the same issue.

**Comment by koalaman** [comment]
The issue was fixed upstream with #1392, and is reflected in v0.8.0 as packaged in ubuntu:22.10. I can reproduce the issue in Docker with ubuntu:20.04 though, and it is indeed annoying. If you want a backport to older Ubuntu packages, you'd have to file a bug on the Ubuntu side.

**Comment by alxndrsn** [comment]
Thanks @koalaman.  Something to look forward to in my next OS update :grin:
